docker/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+# Docker Compose for local development environment
+
+version: "3"
+name: pagopa-interop-public
+volumes:
+  readmodel-volume:
+
+services:
+  # POSTGRESQL to mock AuroraDB
+  readmodel-sql:
```

Comment:
"read model" is the name we use for the read db in the core platform.
To avoid misunderstandings I suggest to define a different name (for this and other occurrences)